students/rgpag/lesson06/test_mailroom.py

```diff
@@ -0,0 +1,63 @@
+import datetime
+import os
+from mailroom import (don_in, donors, new_donor, update_don, menu_sel_1,
+                      menu_sel_2, menu_sel_3, menu_switch_dict)
+
+
+def test_don_in():
+    assert don_in(100) == 100
+
+
+def test_new_donor():
+    assert new_donor('Barack Obama', 1000) == ('Barack Obama', 1, 1000)
+    assert 'Barack Obama' in donors
+    assert donors['Barack Obama'] == [1000]
+
+
+def test_update_don():
+    test_name = 'Mark Zuckerberg'
+    pre_cnt = len(donors[test_name])
+    pre_amt = sum(donors[test_name])
+    assert update_don(test_name, 5000) == (test_name, pre_cnt+1, pre_amt+5000)
+
+
+def test_menu_sel_1():
+    # check new donor
+    assert menu_sel_1('Robbie', 100) == ('Robbie', 1, 100)
+    # check returning donor
+    assert menu_sel_1('Robbie', 100) == ('Robbie', 2, 200)
+    assert menu_sel_1('menu') is False
+    x = 'Jeff Bezos\nMark Zuckerberg\nBill Gates\nPaul Allen\nRobbie'
+# not sure if this approach is what I want to do here...
+    assert menu_sel_1('list') == print(x)
```

Comment:
same here, this also has dependency on `input` function which will wait for user input (not something that can be part of unit tests)